config/300-sks.yaml

```diff
@@ -23,8 +23,8 @@ spec:
   version: v1alpha1
   names:
     kind: ServerlessService
-    plural: sks
-    singular: sks
+    plural: serverlessservices
```

Comment:
Sally sells seashells by the sea shore.

Reply:
Yeah there's a russian one like that :-)